src/AppBundle/Controller/DefaultController.php

```diff
@@ -34,4 +36,59 @@ public function loginAction(Request $request)
             'form'  => $form->createView(),
         ]);
     }
+
+    /**
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function swaggerUiAction()
+    {
+        $docUrl = $this->get('service_container')->getParameter('app_swagger_ui_resource_url');
+
+        if (preg_match('/^(https?:)?\/\//', $docUrl)) {
```

Comment:
It's too hard to understand what happen here. Please contact with me for refactoring this code